### PR TITLE
[Backport] 8268151, 8268643, 8266518, 8268293, 8269031, 8269260, 8265907, 8269955, 8269568, 8269952

### PR DESCRIPTION
**Pull request by JinZhonghui**
[Backport] 8268151, 8268643, 8266518, 8268293, 8269031, 8269260, 8265907, 8269955, 8269568, 8269952
                8269580 and 8269775 not introduced due to the related changes not exist in Dragonwell.

### CODE REVIEW

**Comment by JinZhonghui** [comment]
@JoshuaZhuwj  Please help review it.

**Comment by JoshuaZhuwj** [comment]
> @JoshuaZhuwj Please help review it.

Yes. I'm reviewing. Thanks.

**Review by JoshuaZhuwj — COMMENTED** [review]
Please take a look at my review comments inline.

I also find there exist several issues related to 8269260.

8269179: Crash in TestMacroLogicVector::testSubWordBoolean: assert(_base >= VectorMask && _base <= VectorZ) failed: Not a Vector
8269825: [TESTBUG] Missing testing for x86 KNL platforms
8269828: corrections in some instruction patterns for KNL x86 platform
8269528: VectorAPI Long512VectorTest fails on X86 KNL target

File: test/jdk/jdk/incubator/vector/Vector64ConversionTests.java [hunk 1]
```diff
@@ -47,6 +50,17 @@
...
+ * @test VectorConversionPPC64
+ * @bug 8256479
+ * @requires os.arch == "ppc64" | os.arch == "ppc64le"
+ * @summary VectorConversion on PPC64 without Vector Register usage
+ * @modules jdk.incubator.vector
+ * @modules java.base/jdk.internal.vm.annotation
+ * @run testng/othervm/timeout=1800  -XX:-SuperwordUseVSX -XX:-TieredCompilation --add-opens jdk.incubator.vector/jdk.incubator.vector=ALL-UNNAMED
+ * Vector64ConversionTests
+ */
+
```
[the remark this answers is not in the document]
Comment (JinZhonghui) [reply test/jdk/jdk/incubator/vector/Vector64ConversionTests.java:63]: Op_ClearArray related changes not introduced in 8262355, so 8269828 will not be introduced also.

[the remark this answers is not in the document]
Comment (JinZhonghui) [reply test/jdk/jdk/incubator/vector/Vector64ConversionTests.java:63]: The change of 8269179 is based on 8267652, but 8267652 is not introduced in Dragonwell.

[the remark this answers is not in the document]
Comment (JoshuaZhuwj) [reply test/jdk/jdk/incubator/vector/Vector64ConversionTests.java]: > The change of 8269179 is based on 8267652, but 8267652 is not introduced in Dragonwell.

No. 8267652 exists in Dragonwell 11.
See the below 2 commits in Dragonwell 11.

commit b405eea1c4fa77b2f1aa69e04021fa4ed162f33b (HEAD, zhonghui/JEP414-105, JEP414-105)
Author: Joshua Zhu <yueshi.zwj@alibaba-inc.com>
Date:   Fri Dec 1 11:24:44 2023 +0800

    [Backport] 8267652: c2 loop unrolling by 8 results in reading memory past array

    Summary: Although 8267652 has already been ported in OpenJDK11,
    OpenJDK11 has no vpternlog_mem() which exists in Dragonwell11. Hence
    I have to introduce the missing fix of vpternlog_mem() in 8267652.

    Test Plan: ci jtreg

commit 2c4ab7689898806dcbfe7054babf7d4934f319a3
Author: Vladimir Kempik <vkempik@openjdk.org>
Date:   Mon Oct 18 07:06:27 2021 +0000

    8267652: c2 loop unrolling by 8 results in reading memory past array

    Reviewed-by: mdoerr
    Backport-of: 0c99b192588b04aceaa27cda9cb42b45f95adffe

File: src/hotspot/cpu/x86/vm_version_x86.cpp [hunk 2]
```diff
@@ -743,6 +743,11 @@ void VM_Version::get_processor_features() {
   if( is_intel() ) { // Intel cpus specific settings
     if (is_knights_family()) {
       _features &= ~CPU_VZEROUPPER;
+      _features &= ~CPU_AVX512DQ;
+      _features &= ~CPU_AVX512_VPOPCNTDQ;
+      _features &= ~CPU_AVX512_VPCLMULQDQ;
+      _features &= ~CPU_AVX512_VBMI;
+      _features &= ~CPU_AVX512_VBMI2;
```
[the remark this answers is not in the document]
Comment (JinZhonghui) [reply src/hotspot/cpu/x86/vm_version_x86.cpp:751]: Done.

**Pull request merged**